akka-remote/src/main/scala/akka/remote/transport/AkkaProtocolTransport.scala

```diff
@@ -446,7 +446,7 @@ private[remote] class ProtocolStateActor(
           stop(FSM.Failure(ForbiddenUidReason))
 
         case Associate(handshakeInfo) =>
-          failureDetector.heartbeat()
+//          failureDetector.heartbeat()
```

Comment:
Was this intentionally part of this PR?

Reply:
hah no, that was making sure i had the right failure

Reply:
(it wasn't this one)